src/release-notes.md

```diff
@@ -8,6 +8,28 @@ description: See what's changed as Gradebook evolves!
 
 _Note that we only update our release notes when there is a notable feature or improvement. Between the documented releases here, we can have several smaller releases with bug fixes or improvements_ 😊
 
+## 3.26.2
+
+_Released August 16, 2022_
+
+_Even though there aren't notable features or improvements, we've decided to include the changes from 3.26.0 to 3.26.2 due to reliability issues._
+
+## Bug Fixes
+
+1. Importing a course should now be fully functional
+
+ - Loading the import page will not crash
+
+ - Trying to import a course after logging in will not cause a crash
+
+2. Logging in to Gradebook as a student in a different school will work
```

Comment:
```suggestion
2. Logging in to a school's site on Gradebook will properly take you to the correct school
```

Reply:
a `different` school on Gradebook will take you to `your` correct school?

Reply:
Whatever you decide, please do a quick UI review before merging!

Reply:
I like your wording